### DISCUSSION TITLE
Planning for the release of v0.1.3

### DISCUSSION THREAD

**Discussion by vinayak25** [intentjs/core#5] [Ideas]
Hey folks

I am creating this thread to start a discussion around the next release of v0.1.3 of Intent. Would love to know your thoughts on this.

Features to release:
1. Database Driver for Queues - a small scale db based message queue.
2. Azure driver integration for Storage
3. Support for Guards, Controllers, & Middlewares in the documentation.

These are just few items that I am planning to include in this release, would love to know everyone else's thought on this. Feel free to reply if you would like to see something as part of Intent.

**Comment by JensRoland** [comment]
One thing I would love to see is an optional integration with [Wing](https://www.winglang.io/) -- Intent's 'batteries included' philosophy fits well with Wing's cloud primitives and local simulator.

In an ideal world, you could even have a CLI command create an Intent Queue and the corresponding Wing Queue in one go.

**Reply by vinayak25** [reply]
Looks promising! Will definitely check it out. Though we already have some features in progress. I will revisit this once done with those. Thanks @JensRoland
